src/main/java/com/karankumar/booksapi/resolver/QueryResolver.java

```diff
@@ -44,6 +48,10 @@ public QueryResolver(AuthorRepository authorRepository, BookRepository bookRepos
     public Iterable<Author> findAllAuthors() {
         return authorRepository.findAllAuthors();
     }
+    
+    public List<Book> findByGenre(BookGenre genre) {
+    	return bookRepository.findByGenre(genre);
+
```

Comment:
It may be failing because you're missing the closing brace here. I don't think this will compile

Reply:
fixed that